docker-compose.yml

```diff
@@ -1,6 +1,21 @@
 version: '3'
 services:
+  test:
+    build: .
+    command: 'npm run test:docker:ci'
+    depends_on:
+      - rediscluster
+      - rabbitmq
+    volumes:
+      - .:/usr/windbreaker
+      - /usr/windbreaker/node_modules
+  rediscluster:
+    image: 'grokzen/redis-cluster'
+    ports:
+      - "7000:7000"
+      - "7001:7001"
   rabbitmq:
     image: 'rabbitmq:3.6.10-alpine'
     ports:
       - '5672:5672'
```

Comment:
Do these ports still need to be exposed, or is this just for debugging purposes?